nav/nav_stack/config/costmap_common_params.yaml

```diff
@@ -1,30 +1,4 @@
 max_obstacle_height: 3.3 # Can be a large number, currently matched to the obstacle range
 footprint: [[-0.734, -0.582], [-0.734, 0.582], [0.734, 0.582], [0.734, -0.582]]
-inflation_radius: 0.4
-
-observation_sources: laser_scan_sensor point_cloud_sensor
 
 # NOTE: Caffeine is generally parallel to the ground with 3-wheel drive + battery HACK
-
-laser_scan_sensor: {
```

Comment:
Is there any reason this is deleted and copied into the two files? It would be cleaner if you can move common functionality into a common file

Reply:
I wasn't sure if having the sensors declared for both the global and local cost map would conflict with the SLAM map that will be generated from the first run of the track at the competition in the future. However, I agree in that right now the code is redudant at the moment and I can move it to the common params.

Reply:
Ah ok, you should try it out the other way and see what happens, but I guess it doesn't hurt to do it the way you have it rn since it's only 2 places

Reply:
I feel like keeping them separate for now would make it easier to play around with the global cost map parameters that need to be explored in a separate issue.

Reply:
Resolved by keeping the copied code in both local and global cost map files.